documentation/library_guide/dynamic_selection_api/round_robin_policy.rst

```diff
@@ -0,0 +1,215 @@
+round_robin_policy
+##################
+
+Cycles through the set of resources at each selection. ``round_robin_policy`` 
+is useful for offloading kernels of similar cost to devices of similar
+capabilities. In those cases, a round-robin assignment of kernels to devices
+will achieve a good load balancing.
+
+.. code:: cpp
+
+  namespace oneapi::dpl::experimental {
+  
+    template<typename Backend=sycl_backend> 
+    class round_robin_policy {
+    public:
+      // useful types
+      using resource_type = typename Backend::resource_type;
+      using wait_type = typename Backend::wait_type;
+      
+      class selection_type {
+      public:
+        round_robin_policy<Backend> get_policy() const;
+        resource_type unwrap() const;
+      };
+      
+      // constructors
+      round_robin_policy(deferred_initialization_t);
+      round_robin_policy();
+      round_robin_policy(const std::vector<resource_type>& u);  
+  
+      // deferred initializer
+      void initialize();
+      void initialize(const std::vector<resource_type>& u);
+                      
+      // queries
+      auto get_resources() const;
+      auto get_submission_group();
+      
+      // other implementaton defined functions...
+    };
+  
+  }
+  
+This policy can be used with all of the dynamic selection functions, such as ``select``, ``submit``,
+and ``submit_and_wait``. It also can used with ``policy_traits``.

```

Comment:
Please change to "It can also be used with ``policy_traits``."

Reply:
```suggestion
and ``submit_and_wait``. It can also used with ``policy_traits``.
```